CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ endif()
 find_package(ament_cmake_python REQUIRED)
 find_package(ament_cmake_ros REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
```

Comment:
I think this call to `find_package(rclcpp_lifecycle REQUIRED)` should be under the `BUILD_TESTING` section and only be used when building tests.